spark/src/test/scala/org/apache/spark/sql/delta/ConvertToDeltaSuiteBase.scala

```diff
@@ -419,7 +419,7 @@ trait ConvertToDeltaSuiteBase extends ConvertToDeltaSuiteBaseCommons
       q.stop()
 
       // Add non-streaming data: this should not be ignored in conversion.
-      spark.range(11, 21).select('id.cast("int") as 'col1)
+      spark.range(11, 21).select('id.cast("int") as "col1")
```

Comment:
If we're trying to get rid of deprecated `Symbol` use, shouldn't that be
```suggestion
      spark.range(11, 21).select(col("id").cast("int") as "col1")
```
(bunch more below)

Or is `'foo` only deprecated for passing as an arg to `as` method?

Reply:
yes, it is only for `as` and `subquery`